### PR TITLE
publish cross platform artifacts for PRs

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Build cross platform binaries and publish artifacts as part of PR builds to make it easier testing changes

### CODE REVIEW

**Review by ulyssessouza — CHANGES_REQUESTED** [review]
I think we should have any mechanism to trigger that and not just generate the artifacts for all the PRs.

Maybe using something like the "/test-aci" tag to enable tests on ACI, for example.
A `/generate-artifacts` would be nice!

**Comment by ndeloof** [comment]
@ulyssessouza makes sense.
unfortunately, this makes it harder to retrieve the matching GA workflow run to get binaries

**Comment by ndeloof** [comment]
Added an extra step to update the PR comment with a link to workflow run page, so that user can access built binaries
see sample run on https://github.com/ndeloof/compose-cli/pull/4

File: .github/workflows/pending_answered.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Continuous integration
+name: Remove pending lable on answer
```
Comment (mat007) [inline .github/workflows/pending_answered.yml:1]:
```suggestion
name: Remove pending label on answer
```

File: .github/workflows/artifacts.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        with:
+          name: docker-compose-windows-amd64.exe
+          path: ${{ github.workspace }}/bin/docker-compose-windows-amd64.exe
+
+      - name: Update comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ github.event.comment.id }}
+          body: |
+            This PR can be tested using [binaries](https://github.com/docker/compose-cli/actions/runs/${{ github.run_id }})
```
Comment (mat007) [inline .github/workflows/artifacts.yml:56]:
```suggestion
            This PR can be tested using [binaries](https://github.com/docker/compose-cli/actions/runs/${{ github.run_id }}).
```

**Review by mat007 — APPROVED** [review]
Nice!

**Pull request merged**